dev/devicelab/bin/tasks/hot_mode_dev_cycle_ios_simulator.dart

```diff
@@ -16,7 +16,7 @@ Future<void> main() async {
       await testWithNewIOSSimulator('TestHotReloadSim', (String deviceId) async {
         simulatorDeviceId = deviceId;
         // This isn't actually a benchmark test, so do not use the returned `benchmarkScoreKeys` result.
-        await createHotModeTest(deviceIdOverride: deviceId, checkAppRunningOnLocalDevice: true)();
+        await createHotModeTest(deviceIdOverride: deviceId)();
```

Comment:
Validated this test locally since it won't run in presubmit:
```
Task result:
{
  "success": true,
  "data": null,
  "detailFiles": [],
  "benchmarkScoreKeys": [],
  "reason": "success"
}
```